src/main/java/io/xpire/commons/core/Messages.java

```diff
@@ -5,12 +5,13 @@
  */
 public class Messages {
 
-    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command.";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_ITEM_DISPLAYED_INDEX = "The item index provided is invalid";
+    public static final String MESSAGE_INVALID_ITEM_DISPLAYED_INDEX = "The item index provided is invalid.";
     public static final String MESSAGE_ITEMS_LISTED_OVERVIEW = "%d items listed!";
-    public static final String MESSAGE_INVALID_REMINDER_THRESHOLD = "%s is not a valid reminder threshold";
+    public static final String MESSAGE_INVALID_REMINDER_THRESHOLD = "%s is not a valid reminder threshold.";
     public static final String MESSAGE_INVALID_TAGS = "Tags are not in the item specified!";
-    public static final String MESSAGE_UNKNOWN_DELETE_MODE = "Unknown Delete mode";
+    public static final String MESSAGE_UNKNOWN_DELETE_MODE = "Unknown Delete mode.";
+    public static final String MESSAGE_SUGGESTIONS = " Did you perhaps mean %s?";
```

Comment:
Maybe you could change the message to "Did you mean..." instead?

Reply:
Noted!